templates/docs/examples/patterns/grid/grid-breakout-no-aside.html

```diff
@@ -0,0 +1,19 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Grid / Wide - no aside{% endblock %}
```

Comment:
Please rename it, as it's not Grid example anymore ("Fluid breakout / ..."?).

Also, because it's not a pattern move the file from `/templates/docs/examples/patterns/grid` to `/templates/docs/examples/layouts/...`.

Reply:
done